Corona-Warn-App/src/main/java/de/rki/coronawarnapp/appconfig/mapping/CWAConfigMapper.kt

```diff
@@ -30,11 +30,23 @@ class CWAConfigMapper @Inject constructor() : CWAConfig.Mapper {
             else -> supportedCountriesList
         }
 
+    @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
+    internal fun ApplicationConfigurationAndroid.mapAppFeatures(): List<AppFeaturesOuterClass.AppFeature> =
+        if (!hasAppFeatures()) {
```

Comment:
very minor: Personally, I would prefer a non-negated if statement for easier readability